account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py

```diff
@@ -385,6 +385,13 @@ class OverdueReminderStep(models.TransientModel):
     def _reminder_type_selection(self):
         return self.env["overdue.reminder.action"]._reminder_type_selection()
 
+    # fix bug:
+    # get_invoices() has to be use in mail template
+    # bug: In the wizard after delete an invoice to remind.
+    # The residuals values  are set to 0 in the generated mail.
```

Comment:
Can you put more explanation ?
We spend time to understand it, it's important for other to understand the issue when migrating the module.

Maybe you can put something like

```
# When removing a invoice in the interface the field "mail_body" is recomputed in "onchange mode".
# This mean that the field "invoice_ids" have New Record (NewId) instead of real record.
# The issue is that the field "amount_residual" do not work with New Record
# due to "if line.id" here  https://github.com/odoo/odoo/blob/cc0060e889603eb2e47fa44a8a22a70d7d784185/addons/account/ models/account_move.py#L4029)

# And so the amount_residual will be always be recomputed by the onchange to 0 

# To avoid this issue we force in the template to use real invoices 
```

Reply:
ok, great description, done